File: src/main/java/at/hannibal2/skyhanni/config/features/rift/area/mirrorverse/MirrorVerseConfig.java
package at.hannibal2.skyhanni.config.features.rift.area.mirrorverse;

import at.hannibal2.skyhanni.config.features.rift.area.mirrorverse.danceroomhelper.DanceRoomHelperConfig;
import com.google.gson.annotations.Expose;
import io.github.notenoughupdates.moulconfig.annotations.Accordion;
import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;

public class MirrorVerseConfig {

    // Four Levers

    @ConfigOption(name = "Lava Maze", desc = "")
    @Accordion
    @Expose
    public LavaMazeConfig lavaMazeConfig = new LavaMazeConfig();

    @ConfigOption(name = "Crafting Room", desc = "")
    @Accordion
    @Expose
    public CraftingRoomConfig craftingRoom = new CraftingRoomConfig();

    @ConfigOption(name = "Upside-Down Parkour", desc = "")
    @Accordion
    @Expose
    public UpsideDownParkourConfig upsideDownParkour = new UpsideDownParkourConfig();

    // Red-Green Puzzle

    @ConfigOption(name = "Dance Room Helper", desc = "")
    @Accordion
    @Expose
    public DanceRoomHelperConfig danceRoomHelper = new DanceRoomHelperConfig();

    @ConfigOption(name = "Tubulator", desc = "")
    @Accordion
    @Expose
    public TubulatorConfig tubulatorConfig = new TubulatorConfig();
}

File: src/main/java/at/hannibal2/skyhanni/config/features/rift/area/mirrorverse/MirrorVerseConfig.kt
package at.hannibal2.skyhanni.config.features.rift.area.mirrorverse

import at.hannibal2.skyhanni.config.features.rift.area.mirrorverse.danceroomhelper.DanceRoomHelperConfig
import com.google.gson.annotations.Expose
import io.github.notenoughupdates.moulconfig.annotations.Accordion
import io.github.notenoughupdates.moulconfig.annotations.ConfigOption

class MirrorVerseConfig {
    // Four Levers
    @ConfigOption(name = "Lava Maze", desc = "")
    @Accordion
    @Expose
    val lavaMazeConfig: LavaMazeConfig = LavaMazeConfig()

    @ConfigOption(name = "Crafting Room", desc = "")
    @Accordion
    @Expose
    val craftingRoom: CraftingRoomConfig = CraftingRoomConfig()

    @ConfigOption(name = "Upside-Down Parkour", desc = "")
    @Accordion
    @Expose
    val upsideDownParkour: UpsideDownParkourConfig = UpsideDownParkourConfig()

    // Red-Green Puzzle
    @ConfigOption(name = "Dance Room Helper", desc = "")
    @Accordion
    @Expose
    val danceRoomHelper: DanceRoomHelperConfig = DanceRoomHelperConfig()

    @ConfigOption(name = "Tubulator", desc = "")
    @Accordion
    @Expose
    val tubulatorConfig: TubulatorConfig = TubulatorConfig()
}
